python-service-identity (18.1.0-8) unstable; urgency=medium

  * Cherry-pick addition of missing six dependency from upstream
    (Closes: #1030628).

 -- Robie Basak <robie@justgohome.co.uk>  Sun, 05 Feb 2023 22:10:42 +0000

python-service-identity (18.1.0-7) unstable; urgency=medium

  [ Ondřej Nový ]
  * d/control: Update Maintainer field with new Debian Python Team
    contact address.
  * d/control: Update Vcs-* fields with new Debian Python Team Salsa
    layout.

  [ Debian Janitor ]
  * Bump debhelper from old 12 to 13.

 -- Sandro Tosi <morph@debian.org>  Mon, 20 Jun 2022 01:41:00 -0400

python-service-identity (18.1.0-6pexip4) pexip-bullseye; urgency=medium

  * Bullseye rebuild for ARM64 inclusion

 -- Mark Hymers <mark.hymers@pexip.com>  Mon, 05 Sep 2022 10:35:39 +0100

python-service-identity (18.1.0-6pexip3) pexip-bullseye; urgency=medium

  * Fix up debian/control Maintainer/Uploaders fields.

 -- Mark Hymers <mark.hymers@pexip.com>  Tue, 28 Jun 2022 17:44:40 +0100

python-service-identity (18.1.0-6pexip2) pexip-bullseye; urgency=medium

  * Main bullseye build

 -- Mark Hymers <mark.hymers@pexip.com>  Fri, 24 Jun 2022 14:04:32 +0100

python-service-identity (18.1.0-6pexip1) pexip; urgency=medium

  * Initial bullseye build

 -- Mark Hymers <mark.hymers@pexip.com>  Thu, 02 Jun 2022 14:42:50 +0100

python-service-identity (18.1.0-6) unstable; urgency=medium

  * Drop python2 support; Closes: #938164

 -- Sandro Tosi <morph@debian.org>  Wed, 01 Apr 2020 21:09:17 -0400

python-service-identity (18.1.0-5~bpo10+1pexip1) pexip; urgency=medium

  * Migrate to Buster Backports

 -- Huw Jones <huw@pexip.com>  Mon, 11 Jan 2021 17:35:40 +0000

python-service-identity (18.1.0-5) unstable; urgency=medium

  * No-change rebuild.

 -- Andrej Shadura <andrewsh@debian.org>  Sat, 13 Jul 2019 01:16:30 +0200

python-service-identity (18.1.0-4) unstable; urgency=medium

  * Don’t *actually* fail when there are no tests to run.

 -- Andrej Shadura <andrewsh@debian.org>  Thu, 11 Jul 2019 20:06:23 +0200

python-service-identity (18.1.0-3) unstable; urgency=medium

  * Skip tests when not present in the upstream tarball.

 -- Andrej Shadura <andrewsh@debian.org>  Wed, 26 Jun 2019 12:59:52 -0500

python-service-identity (18.1.0-2) unstable; urgency=medium

  * Upload to unstable (Closes: #930636).
  * Use debhelper-compat 12.
  * Update copyright years.
  * Bump Standards-Version to 4.3.0.3 (no changes).

 -- Andrej Shadura <andrewsh@debian.org>  Mon, 24 Jun 2019 15:30:09 -0500

python-service-identity (18.1.0-1) experimental; urgency=medium

  [ Ondřej Nový ]
  * d/control: Set Vcs-* to salsa.debian.org
  * d/copyright: Use https protocol in Format field

  [ Andrej Shadura ]
  * Upload to experimental.
  * New upstream version.
  * Add new build dependencies.
  * Bump debhelper from old 9 to 10.
  * Re-export upstream signing key without extra signatures.

 -- Andrej Shadura <andrewsh@debian.org>  Fri, 21 Jun 2019 08:52:20 -0500

python-service-identity (16.0.0-2) unstable; urgency=medium

  * Adopt package; thanks to Matthias Klose for his work so far.
  * Update debian/watch to check signatures and use https.
  * Switch python{,3}-characteristic dependency to python{,3}-attr
    (closes: #815385).

 -- Tristan Seligmann <mithrandi@debian.org>  Sun, 21 Feb 2016 23:30:43 +0200

python-service-identity (16.0.0-1) unstable; urgency=medium

  * New upstream release.
  * Relax build dependencies. Closes: #799687.

 -- Matthias Klose <doko@debian.org>  Sat, 20 Feb 2016 22:23:25 +0100

python-service-identity (14.0.0-1) unstable; urgency=medium

  * New upstream release.
  * Build-depend on python3-idna, python-idna.

 -- Matthias Klose <doko@debian.org>  Wed, 02 Sep 2015 21:17:52 +0200

python-service-identity (1.0.0-3) unstable; urgency=medium

  * Drop build dependency on python{,3}-idna (needs some more packaging work).

 -- Matthias Klose <doko@debian.org>  Sat, 09 Aug 2014 21:02:48 +0200

python-service-identity (1.0.0-2) unstable; urgency=medium

  * Enable running the testsuite during the build.

 -- Matthias Klose <doko@debian.org>  Sun, 03 Aug 2014 19:44:42 +0200

python-service-identity (1.0.0-1) unstable; urgency=low

  * Initial release (Closes: #754866).

 -- Matthias Klose <doko@debian.org>  Tue, 15 Jul 2014 09:35:43 +0000
